maintainers/maintainer-list.nix

```diff
@@ -5853,6 +5853,11 @@
     githubId = 17859309;
     name = "Fuzen";
   };
+  fuzzdk = {
```

Comment:
Put this in a separate commit (no need for a new PR).

Reply:
done